tests/foreman/ui/test_rhcloud_insights.py

```diff
@@ -287,6 +286,7 @@ def test_host_details_page(
             silent_failure=True,
             handle_exception=True,
         )
+        session.cloudinsights.sync_hits()
```

Comment:
We have `wait_for` before this line, which waits for sync task to complete.
So it doesn't look like the right fix to me.

Reply:
Somehow the remediations are not following after the full inventory sync which is resulting in an assertion failure
for `assert int(result['Recommendations']) > 0` hence I am trying to sync recommendations after the complete inventory sync.

Reply:
so what I am seeing is @jameerpathan111 removal of the wait_for function is not impacting the sync_hits for which we had the timeout hence so removing it wont be a issue and test is passing as well.

Reply:
yeah if sync is instantaneous it shouldn't be an issue but if it's not then the test will fail.
But we can give it a try and see how it goes.

Reply:
Passed on PRT as well!